.github/workflows/pull-request.yml

```diff
@@ -24,6 +24,7 @@ jobs:
         "packages/paywall",
         "smart-contracts",
         "subgraph",
+        "tests",
```

Comment:
this adds the `tests` folder in the CI check so we still run integration tests even if there are changes only in that folder

Reply:
so that doesnt workas it is  as CI attempts to launch it as unit test

Reply:
We really need a better way to detect changes...